discord-scripts/thread-management/check-thread-archiving.ts

```diff
@@ -31,7 +31,7 @@ import {
 
 // The maximum time between any two messages after which a thread is considered
 // async.
-const MAX_HEURISTIC_SYNC_THREAD_DURATION = 60 * MINUTE // 60 * MINUTE
+const MAX_HEURISTIC_SYNC_THREAD_DURATION = 12 * HOUR // 60 * MINUTE
```

Comment:
Let's hold off on this change; it's predicated on the assumption that a quick thread is acceptable,  but 12 hours really isn't a quick thread anymore. 

Let's see what happens if we just fix the other issue first.